CONTRIBUTING.md

```diff
@@ -46,6 +46,35 @@ If you want to contribute your own package to openwhisk-catalog, you need to hav
 - `installYOURPKG.sh` to install your package. `YOURPKG` can be the name of your package.
 - Add the invocation of your package install script to `installCatalog.sh`.
 
+### How to write the tests for your package
+
+You must provide the test codes for your package. OpenWhisk catalog follows the same coding style of OpenWhisk. OpenWhisk recommends to use [scalatest](http://www.scalatest.org) in the test codes.
+
+#### Base classes for test
+
+OpenWhisk has defined the base classes to formalize the test codes in the folder of openwhisk/tests/src/common. Generally, your tests should extend following classes:
+- [TestHelpers](https://github.com/openwhisk/openwhisk/blob/master/tests/src/common/TestHelpers.scala) is the abstract base class to define the common features OpenWhisk uses most: FlatSpec and BeforeAndAfterEachTestData.
+- [WskTestHelpers](https://github.com/openwhisk/openwhisk/blob/master/tests/src/common/WskTestHelpers.scala) is a test fixture to ease cleaning of whisk entities created during testing. By using functions `withActivation` and `withAssetCleaner` in the test codes, you don't need to care about the entities you create during test.
+
+You may want to use the following utilities to ease the unit test programing.
+- [Wsk](https://github.com/openwhisk/openwhisk/blob/master/tests/src/common/Wsk.scala) provides Scala bindings for the whisk CLI. It defines utility classes for top level CLI commands, e.g. `WskAction` for `wsk action`, `WskTrigger` for `wsk trigger` and etc, including some methods for the easily invocation. For example, if you want to invoke an action, you can simply implement by the following codes:
+```
+val wsk = new Wsk(usePythonCLI=true)
```

Comment:
We do not use the python cli anymore.
We can directly use go cli by defining val wsk = new Wsk()